reference/request-bodies/transaction_create_request.yaml

```
@@ -50,12 +50,45 @@ properties:
   payment_method:
     x-model-name: TransactionPaymentMethodRequest
     oneOf:
-      - $ref: "../openapi.yaml#/components/schemas/CardRequest"
-      - $ref: "../openapi.yaml#/components/schemas/RedirectRequest"
+      - type: object
+        allOf:
+          - $ref: "../openapi.yaml#/components/schemas/CardRequest"
+        properties:
+          external_identifier:
+            type: string
+            description:
+              An external identifier that can be used to match the card
+              against your own records. This can only be set if the `store`
+              flag is set to `true`.
+            example: card-323444
+            nullable: true
+      - type: object
+        allOf:
+          - $ref: "../openapi.yaml#/components/schemas/RedirectRequest"
+        properties:
+          external_identifier:
+            type: string
+            description:
+              An external identifier that can be used to match the account
```

I copied this over from the `RedirectRequest` schema, but should this actually be
```suggestion
              An external identifier that can be used to match the card
```
?

---

The redirect payment method does not necessarily reference a card underneath, so I guess that this is referring to PayPal accounts and so on.